drivers/serial/Kconfig.emul

```diff
@@ -7,6 +7,8 @@ config UART_EMUL
 	bool "Emulated UART driver [EXPERIMENTAL]"
 	default y
 	depends on DT_HAS_ZEPHYR_UART_EMUL_ENABLED
+	depends on EMUL
```

Comment:
Wonder if `select` would be better here. Then you won't have to add `CONFIG_EMUL=y` in the test config files.

Reply:
I did the same as for the other emulated bus devices.